beam-postgres/test/Database/Beam/Postgres/Test/Migrate.hs

```diff
@@ -70,3 +77,15 @@ charNoWidthVerification pgConn tyName charTy =
           case res of
             VerificationSucceeded -> return ()
             VerificationFailed failures -> fail ("Verification failed: " ++ show failures)
+
+-- | Verifies that 'verifySchema' correctly checks enabled PgCrypto extension
+extensionVerification :: IO ByteString -> TestTree
+extensionVerification pgConn =
+    testCase "verifySchema correctly checks enabled PgCrypto extension" $
+      withTestPostgres "db_extension_pgcrypto" pgConn $ \conn ->
+        runBeamPostgres conn $ do
+          db <- executeMigration runNoReturn (CryptoDb <$> pgCreateExtension)
+          res <- verifySchema migrationBackend db
```

Comment:
Seems like we should run `verifySchema` both before and after migration to ensure it's not a false negative?